src/lib/iframe-tools/identity-manager.js

```diff
@@ -87,12 +87,28 @@ export class IdentityManager extends IframeManager {
       return;
     }
 
-    debugLogger.log('[identity-manager:fetchIdentity] Fetch country request.');
-    this.fetchCountry().then((country) => {
+    let shouldReallyCheckCountry;
+    const localStorageKey = 'pp-sdk-shouldReallyCheckCountry';
+    const LS = window.localStorage;
+    if (LS.getItem(localStorageKey) === null) {
+      const randomChance = Math.floor(Math.random() * 100);
+      shouldReallyCheckCountry = randomChance < 50;
+      LS.setItem(localStorageKey, shouldReallyCheckCountry);
+    } else {
+      shouldReallyCheckCountry = LS.getItem(localStorageKey);
+    }
 
-      debugLogger.log('[identity-manager:fetchIdentity] Fetch identity request. Country:', country);
-      this.fetchUserInfo(country);
-    });
+    debugLogger.log(`[identity-manager:fetchIdentity] Fetching Identity`);
+
+    if (shouldReallyCheckCountry) {
```

Comment:
I think we could also remove the need for `shouldReallyCheckCountry` if the suggestion above is followed: https://github.com/paypal/paypal-muse-components/pull/159/files#r819142163.  The if could just check `LS.getItem(localStorageKey)`.  Also `localStorageKey` could use a more specific name like `shouldReallyCheckCountryKey`.

Reply:
- `shouldCheckCountry` makes the condition more readable.
- There is a `constants.js` which has the keys used with localStorage .... however this code is to start a temporary ramp (currently at 50%) and will be removed in the next sprint once it's at 100%.